.gitignore

```diff
@@ -1,2 +0,0 @@
-node_modules/
-.env 
```

Comment:
Readme file should contain details about the project and how to run a project, can you please add those

Reply:
ok